logic/peer.js

```diff
@@ -30,11 +30,121 @@ const ip = require('ip');
  * @returns Calls accept method
  * @todo Add description for the params
  */
-// Constructor
-function Peer(peer) {
-	return this.accept(peer || {});
+class Peer {
+	constructor(peer) {
+		return this.accept(peer || {});
+	}
+
+	/**
+	 * Checks peer properties and adjusts according rules.
+	 *
+	 * @param {peer} peer
+	 * @returns {Object} this
+	 * @todo Add description for the params
+	 */
+	accept(peer) {
```

Comment:
How come we can define these on the class here but elsewhere we can't?

Reply:
At the moment we cant change the implementation of static functions because of the tests and `modules` needs to be changed as well, this becomes too many changes in shot, wherever I had the opportunity to change, like functions with `this` can be moved inside class as the behaviour wont change, I have done that, rest of them I have kept as it is.